seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json

```diff
@@ -165,10 +165,10 @@
       ],
       "solutions": [],
       "hints": [],
-      "type": "basejump",
+      "type": "waypoint",
```

Comment:
I'll amend the commit shortly, old laptop died and I've had to reset up my environments on a new system